EnglishFilter/sections/css_extended.txt

```diff
@@ -3,6 +3,8 @@
 !
 ! https://github.com/easylist/easylist/pull/6896#pullrequestreview-599945327
 cambb.xxx#?#.thumb div.ad:upward(.thumb)
+! https://github.com/AdguardTeam/AdguardFilters/issues/76089
+wpshare.net#?#section[class^="widget wpshare-widget"] > h4[class^="widget-title"]:contains(Advertisement)
```

Comment:
I don't see it. Only dark leftovers `wpshare.net##div[style^="width: 100%;height: 300px;display: flex;"]`

Reply:
Ok go to homepage.

Reply:
This rule `||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,important,domain=wpshare.net` hides dark leftovers already.

Reply:
Hm. Sure.